test/e2e/common/pods.go

```diff
@@ -848,4 +853,115 @@ var _ = framework.KubeDescribe("Pods", func() {
 		validatePodReadiness(false)
 
 	})
+
+	ginkgo.It("should run through the lifecycle of Pods and PodStatus", func() {
+		podResource := schema.GroupVersionResource{Group: "", Version: "v1", Resource: "pods"}
+		testNs := "default"
```

Comment:
metav1.NamespaceDefault,

Reply:
I was meant to switch this out for ``f.Namespace.Name` when creating this PR. Updated in 1e98c1ba6c0